docs/user/container.md

```diff
@@ -455,23 +455,50 @@ Server Checksumming   on
 
 ### Erasure Code
 
-DAOS erasure code implementation uses a fixed cell size that applies to all
-objects in the container. The cell size in DAOS is the size of each data and
-parity fragments (also called sometimes chunks). The cell size can be set at
-container creation time via the property:
+The DAOS erasure code implementation uses a fixed cell size that applies to all
+objects in the container.
+The cell size in DAOS is the size of a single data and parity fragment.
+By default, a container's `ec_cell_sz` property is inherited from the pool's
+default `ec_cell_sz`, which was 1MiB in DAOS 2.0 and has been reduced to
+64kiB in DAOS 2.2.  The container cell size can also be set at
```

Comment:
Very minor I think (unless I am not understanding something).
There are uses of kiB and KiB in the documentation (some of which is output from commands - not sure if there is an inconsistency in command output text)

For the parts of the patch that are written documentation (not pasted command output), does it make sense to use the same case e.g., KiB everwhere versus kIB some places and KiB in other places?